modules/javafx.graphics/src/main/native-glass/mac/a11y/AccessibleBase.m

```diff
@@ -175,6 +177,27 @@ - (id)accessibilityTitleUIElement
     return variantToID(env, jresult);
 }
 
+- (NSArray *)accessibilityChildren
+{
+    jobject jresult = NULL;
+    GET_MAIN_JENV;
+    if (env == NULL) return NULL;
+    jresult = (jobject)(*env)->CallLongMethod(env, self->jAccessible,
+                                              jAccessibilityAttributeValue, (jlong)@"AXChildren");
+    GLASS_CHECK_EXCEPTION(env);
+    return variantToID(env, jresult);
```

Comment:
A general comment(not for this PR): There seems to be many methods with same body except change in last parameter in call to `CallLongMethod`. May be can change these methods to have a helper method.

Reply:
> May be can change these methods to have a helper method.

I will probably do this refactoring in a separate PR.